### PR TITLE
[Doc](benchmark) update 3.0 benchmark doc.

### PR DESCRIPTION
**Pull request by feifeifeimoon**
## Versions 

- [x] dev
- [x] 3.0
- [x] 2.1
- [x] 2.0

## Languages

- [x] Chinese
- [x] English

## Docs Checklist

- [ ] Checked by AI
- [ ] Test Cases Built

### CODE REVIEW

File: i18n/zh-CN/docusaurus-plugin-content-docs/version-3.0/benchmark/ssb.md [hunk 1]
```diff
@@ -30,25 +30,24 @@ under the License.
 
 本文档主要介绍 Apache Doris 在 SSB 1000G 测试集上的性能表现。
 
-在 SSB 标准测试数据集上的 13 个查询上，我们基于 Apache Doris 2.0.6 版本进行了测试。
+在 SSB 标准测试数据集上的 13 个查询上，我们基于 Apache Doris 2.0.16 版本进行了测试。
```
Comment (KassieZ) [inline i18n/zh-CN/docusaurus-plugin-content-docs/version-3.0/benchmark/ssb.md]: 2.0.15.1

File: i18n/zh-CN/docusaurus-plugin-content-docs/version-3.0/benchmark/ssb.md [hunk 2]
```diff
@@ -30,25 +30,24 @@ under the License.
...
 
 ## 2. 软件环境
 
 - Doris 部署 3BE 1FE
-- 内核版本：Linux version 5.4.0-96-generic (buildd@lgw01-amd64-051)
+- 内核版本：Linux version 5.15.0-101-generic 
 - 操作系统版本：Ubuntu 20.04 LTS (Focal Fossa)
-- Doris 软件版本：Apache Doris 2.0.6
-- JDK：openjdk version "1.8.0_131"
+- Doris 软件版本：Apache Doris 2.0.16-rc01
```
Comment (KassieZ) [inline i18n/zh-CN/docusaurus-plugin-content-docs/version-3.0/benchmark/ssb.md]: 2.0.15.1

File: versioned_docs/version-3.0/benchmark/ssb.md [hunk 3]
```diff
@@ -30,25 +30,24 @@ under the License.
 
 This document mainly introduces the performance of Doris on the SSB 1000G test set.
 
-We tested 13 queries on the SSB standard test dataset based on Apache Doris version 2.0.6.
+We tested 13 queries on the SSB standard test dataset based on Apache Doris version 2.0.16.
```
Comment (KassieZ) [inline versioned_docs/version-3.0/benchmark/ssb.md]: 2.0.15.1

File: versioned_docs/version-3.0/benchmark/ssb.md [hunk 4]
```diff
@@ -63,46 +62,46 @@ We tested 13 queries on the SSB standard test dataset based on Apache Doris vers
...
-| q2.3  | 480              |
-| q3.1  | 1339             |
-| q3.2  | 957              |
-| q3.3  | 215              |
-| q3.4  | 34               |
-| q4.1  | 1569             |
-| q4.2  | 174              |
-| q4.3  | 109              |
-| Total | 6696             |
+Here we use Apache Doris 2.0.16 for comparative testing. In the test, we use Query Time(ms) as the main performance indicator. The test results are as follows:
```
Comment (KassieZ) [inline versioned_docs/version-3.0/benchmark/ssb.md]: 2.0.15.1

File: versioned_docs/version-3.0/benchmark/ssb.md [hunk 5]
```diff
@@ -30,25 +30,24 @@ under the License.
...
 
 ## 2. Software Environment
 
 - Doris Deployed 3BEs and 1FE
-- Kernel Version: Linux version 5.4.0-96-generic (buildd@lgw01-amd64-051)
+- Kernel Version: Linux version 5.15.0-101-generic 
 - OS version: Ubuntu 20.04 LTS (Focal Fossa)
-- Doris software version: Apache Doris 2.0.6.
-- JDK: openjdk version "1.8.0_131"
+- Doris software version: Apache Doris 2.0.16-rc01
```
Comment (KassieZ) [inline versioned_docs/version-3.0/benchmark/ssb.md]: 2.0.15.1

File: versioned_docs/version-3.0/benchmark/ssb.md [hunk 6]
```diff
@@ -63,46 +62,46 @@ We tested 13 queries on the SSB standard test dataset based on Apache Doris vers
...
-| q3.2  | 957              |
-| q3.3  | 215              |
-| q3.4  | 34               |
-| q4.1  | 1569             |
-| q4.2  | 174              |
-| q4.3  | 109              |
-| Total | 6696             |
+Here we use Apache Doris 2.0.16 for comparative testing. In the test, we use Query Time(ms) as the main performance indicator. The test results are as follows:
+
+| Query     | Doris 2.0.16 (ms) |
```
Comment (KassieZ) [inline versioned_docs/version-3.0/benchmark/ssb.md]: 2.0.15.1

File: versioned_docs/version-3.0/benchmark/ssb.md [hunk 7]
```diff
@@ -63,46 +62,46 @@ We tested 13 queries on the SSB standard test dataset based on Apache Doris vers
...
-| q3.2  | 944              |
-| q3.3  | 766              |
-| q3.4  | 146              |
-| q4.1  | 3451             |
-| q4.2  | 829              |
-| q4.3  | 325              |
-| Total | 12480            |
+Here we use Apache Doris 2.0.16 for comparative testing. In the test, we use Query Time(ms) as the main performance indicator. The test results are as follows:
+
+| Query     | Doris 2.0.16 (ms) |
```
Comment (KassieZ) [inline versioned_docs/version-3.0/benchmark/ssb.md]: 2.0.15.1

File: versioned_docs/version-3.0/benchmark/ssb.md [hunk 8]
```diff
@@ -63,46 +62,46 @@ We tested 13 queries on the SSB standard test dataset based on Apache Doris vers
...
-| q2.3  | 768              |
-| q3.1  | 2924             |
-| q3.2  | 944              |
-| q3.3  | 766              |
-| q3.4  | 146              |
-| q4.1  | 3451             |
-| q4.2  | 829              |
-| q4.3  | 325              |
-| Total | 12480            |
+Here we use Apache Doris 2.0.16 for comparative testing. In the test, we use Query Time(ms) as the main performance indicator. The test results are as follows:
```
Comment (KassieZ) [inline versioned_docs/version-3.0/benchmark/ssb.md]: 2.0.15.1

**Review by KassieZ — APPROVED** [review]

**Review by wm1581066 — APPROVED** [review]
LGTM

**Pull request merged**